_pages/dev/features/cds-merchandising-carousels.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Context-Driven Merchandising Carousels
+---
+
+**Note:** This feature is part of the Context-Driven Merchandising module introduced in Spartacus version 1.5.
+
+## Overview
+
+Context-Driven Merchandising carousels enable a Spartacus storefront to display the results of Context-Driven Merchandising in the form of a product carousel.
+
+For more details on Context-Driven Merchandising, see [the dedicated documentation](https://help.sap.com/viewer/26c27f420a2946e19aaf1518849f932d/SHIP/en-US/fda52c18718648dcbd57515e7c6fefaf.html).
+
+## Requirements
+
+Context-Driven Merchandising carousels require the `merchandisingaddon` and `profiletagaddon` extensions. Once you install the extensions, follow the steps in [Merchandising Configuration Setup](https://help.sap.com/viewer/50c996852b32456c96d3161a95544cdb/latest/en-US/57bd76612cea4fddb2d62d2b29d0effb.html#loio57bd76612cea4fddb2d62d2b29d0effb) and [Catalog Synchronization Configuration](https://help.sap.com/viewer/50c996852b32456c96d3161a95544cdb/latest/en-US/a13f4af6daa24d66b4c9a2b0e5544160.html#loioa13f4af6daa24d66b4c9a2b0e5544160) to synchronize a product catalog with Context-Driven Services. 
+
+**Note:** If you have SAP Commerce patch version 1905.8 or later, follow the instructions in [Product Category Configuration](https://help.sap.com/viewer/50c996852b32456c96d3161a95544cdb/latest/en-US/a13f4af6daa24d66b4c9a2b0e5544160.html#loio6f59fb60a3fd43f89c08e8ba28b9e2a2).
```

Comment:
Could we update the link text to `Product Directory Configuration` as that is the "sub-section" of the page we're linking to?

Reply:
good catch! fixed!